R/gtrends.R

```diff
@@ -571,7 +586,7 @@ as.zoo.gtrends <- function(x, ...) {
     blocks <- lapply(start:length(vec), function(i)
       read.csv(
         textConnection(strsplit(vec[i], "\\\n")[[1]]),
-        skip = 1,
+        skip = 1, header = FALSE,
```

Comment:
If you do that, you will end up with:

```r
> gtrends(c("NBA","NFL"), geo = "CA")$Top.cities.for.nba
              V1  V2
1           City nba
2       Brampton 100
3        Markham  94
4           Ajax  83
5        Toronto  78
6        Vaughan  76
7  Richmond Hill  76
8    Mississauga  76
9       Richmond  62
10      Waterloo  58
11      Oakville  55
12      Winnipeg  51
13        Surrey  51
14      Hamilton  50
15       Windsor  49
16        Oshawa  44
```
The first row should be treated as columns.